app/webpacker/components/Tickets/TicketWorkbenches/AnonymizationTicketWorkbenchForWrt/VerifyAnonymizeDetails.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { List, Message } from 'semantic-ui-react';
+
+const anonymizationType = (data) => {
+  if (data.user_details && !data.person_details) {
+    return 'Account Only';
+  } if (data.person_details && !data.user_details) {
+    return 'Profile Only';
+  } if (data.user_details && data.person_details) {
+    return 'Account & Profile';
+  }
+  return 'Unknown';
+};
```

Comment:
Just a minor comment, because it's not my power to decide: I am confused about the terminology here. In the entry point to the script (further up, where you have a search input) you call it "User" and "Person". Here, you suddenly talk about "Profile" and "Account". Maybe this PR is a good chance to clean up some terminology?